bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,236 @@
+# Ferroamp Binding
+
+The Ferroamp binding is used to get live data from Ferroamp EnergyHub
+
+The Ferroamp binding is compatible with EnergyHub Wall and EnergyHub XL, and connects to your local EnergyHub via LAN.
+Data and commands are received/sent using MQTT where the user connects to the MQTT broker residing on the EnergyHub.
+The communication with the broker might take some minute to establish, so Please just be patient. The Thing will be
+in state INITIALIZATION and UNKNOWN during this time and then change to state ONLINE once connection is established.
+
+*note* Contact Ferroamp support to enable MQTT in the EnergyHub and to get the Username and Password:
+
+<https://ferroamp.com/om-ferroamp/>
+
+Every Sso has a unique serial number which is marked on the side of the SSO unit. Ex. PS00990-A04-S20120476.
+This number is to identify the respective Sso Pv-string.
+
+## Supported Things
+
+The binding retrieves data from the different parts of the Ferroamp EnergyHub such as:
+
+- `ehub`: EnergyHub Wall and EnergyHub XL.
+- `sso` : Solar string optimizer.
+- `eso` : Bidirectional DC/DC converter for connection of battery.
+- `esm` : Energy Storage Module.
```

Comment:
The binding supports just one  Thing: `energyhub`.
The list suggests these are all thing, but they are not in terms of openHAB. They are moduels/devices that are exposed by the `energyhub`.

Maybe something like:

This binding supports one Thing, the `energyhub`. This hub makes it able to get data from multiple devices: `sso`, `eso`....etc.

Reply:
20250319
Changes done in line 19